src/pyodide/python-entrypoint-helper.ts

```diff
@@ -85,6 +99,14 @@ async function setupPatches(pyodide: Pyodide): Promise<void> {
     // install any extra packages into the site-packages directory
     const sitePackages = pyodide.FS.sitePackages;
 
+    // Expose the doAnImport function and global modules to Python globals
+    // @ts-expect-error: Assign to global object
+    globalThis.pyodide_entrypoint_helper = {
+      doAnImport,
+      cloudflareWorkersModule,
+      cloudflareSocketsModule,
+    };
```

Comment:
As a nit, it'd be mildly preferable to use `pyodide.registerJsModule("_pyodide_entrypoint_helper", {...})` here and then do `from _pyodide_entrypoint_helper import doAnImport`. That would avoid polluting the JS global scope. Then again, it's not like anyone else is using the JS global scope, so this is less important than it'd be for an upstream Pyodide feature.